.github/workflows/awsfulltest.yml

```diff
@@ -23,95 +25,11 @@ jobs:
-          workdir: s3://${{ secrets.AWS_S3_BUCKET }}/work/proteinfold/work-${{ github.sha }}
-          parameters: |
-            {
-              "input": 'https://raw.githubusercontent.com/nf-core/test-datasets/proteinfold/testdata/samplesheet/v1.0/samplesheet_multimer.csv'
-              "outdir": "s3://${{ secrets.AWS_S3_BUCKET }}/proteinfold/results-${{ github.sha }}/multimer_mode_${{ matrix.mode }}"
-              "model_preset": "multimer"
-            }
-          profiles: test_full,aws_tower
-      - uses: actions/upload-artifact@v3
-        with:
-          name: Tower debug log file
-          path: tower_action_*.log
-
-  run-tower-colabfold-monomer:
-    name: Run AWS colabfold full monomer tests
-    if: github.repository == 'nf-core/proteinfold'
-    runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        mode: ["colabfold_webserver", "colabfold_local"]
-    steps:
-      - name: Launch workflow via tower
-        uses: nf-core/tower-action@v3
-        with:
-          workspace_id: ${{ secrets.TOWER_WORKSPACE_ID }}
-          access_token: ${{ secrets.TOWER_ACCESS_TOKEN }}
-          compute_env: ${{ secrets.TOWER_COMPUTE_ENV }}
-          workdir: s3://${{ secrets.AWS_S3_BUCKET }}/work/proteinfold/work-${{ github.sha }}
-          parameters: |
-            {
-              "input": 'https://raw.githubusercontent.com/nf-core/test-datasets/proteinfold/testdata/samplesheet/v1.0/samplesheet.csv'
-              "outdir": "s3://${{ secrets.AWS_S3_BUCKET }}/proteinfold/results-${{ github.sha }}/monomer_mode_${{ matrix.mode }}"
-              "model_type": "AlphaFold2-ptm"
-            }
-          profiles: test_full,aws_tower
-      - uses: actions/upload-artifact@v3
-        with:
-          name: Tower debug log file
-          path: tower_action_*.log
-
-  run-tower-colabfold-multimer:
-    name: Run AWS colabfold full multimer tests
-    if: github.repository == 'nf-core/proteinfold'
-    runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        mode: ["colabfold_webserver", "colabfold_local"]
-    steps:
-      - name: Launch workflow via tower
-        uses: nf-core/tower-action@v3
-        with:
-          workspace_id: ${{ secrets.TOWER_WORKSPACE_ID }}
-          access_token: ${{ secrets.TOWER_ACCESS_TOKEN }}
-          compute_env: ${{ secrets.TOWER_COMPUTE_ENV }}
-          workdir: s3://${{ secrets.AWS_S3_BUCKET }}/work/proteinfold/work-${{ github.sha }}
-          parameters: |
-            {
-              "input": 'https://raw.githubusercontent.com/nf-core/test-datasets/proteinfold/testdata/samplesheet/v1.0/samplesheet_multimer.csv'
-              "outdir": "s3://${{ secrets.AWS_S3_BUCKET }}/proteinfold/results-${{ github.sha }}/multimer_mode_${{ matrix.mode }}"
-              "model_type": "AlphaFold2-multimer-v2"
-            }
-          profiles: test_full,aws_tower
+          profiles: test_full_${{ matrix.platform }}_${{ matrix.structure }},aws_tower
```

Comment:
Here we might have an issue. First we have to change matrix.platform to matrix.mode. On top of this the values in the mode matrix are not consistent with the config filenames

Reply:
Yeah, platform was a typo, I use structure since I use the profiles to call the runs, see [here](https://github.com/JoseEspinosa/nf-core-proteinfold/blob/439cc903d67065deb0ad19f0fe1c512358bbb180/nextflow.config#L187-L198)